python/plugins/processing/modeler/AddModelFromFileAction.py

```diff
@@ -77,7 +77,8 @@ def execute(self):
                     QMessageBox.Yes | QMessageBox.No,
                     QMessageBox.No)
 
-                if reply == QMessageBox.Yes:
-                    shutil.copyfile(filename, destFilename)
+                if reply == QMessageBox.No:
+                    return
 
+            shutil.copyfile(filename, destFilename)
```

Comment:
```suggestion
                shutil.copyfile(filename, destFilename)
```

I would keep this line inside if condition in case destFilename does not exist.